mkdocs.yml

```
@@ -1,10 +1,23 @@
 docs_dir: docs/book
 site_dir: docs/html
 nav:
-    - Home: index.md
-    - Introduction: intro.md
+  - Home: index.md
+  - v2:
+      - Introduction: v2/intro.md
+      - Installation: v2/installation.md
+      - Quick Start: v2/quick-start.md
+      - Routing Middleware: v2/routing-middleware.md
+      - Migration: v2/migration.md
+  - v1:
+      - Introduction: v1/intro.md
```

The URL will change with this update therefore we need a redirect:

```suggestion
  - v1:
      - Introduction: v1/intro.md
  - _hidden-legacy-page-links:
      - _intro: intro.md
```

The content of the old `intro.md` file:

```html
<noscript><meta http-equiv="refresh" content="0; url=/laminas-mvc-middleware/v2/intro/"></noscript>
<script>
  document.addEventListener("DOMContentLoaded", function (event) {
    window.location.pathname = '/laminas-mvc-middleware/v2/intro/';
  });
</script>

```

---

Thanks for the quick reply.
Lots of mkdocs voodoo here 🙂 
Could that redirect stuff and script snipped be extracted into the laminas theme, just like the installation page?

---

Yes, it's not very nice and there is already another solution. I will fix this with the next update of the documentation theme.